editor/src/core/shared/file-utils.ts

```diff
@@ -167,3 +167,8 @@ export function getFileExtension(filename: string): string {
     return '' // TODO How do we handle files with no extension?
   }
 }
+
+export function directoryForFilePath(path: string): string {
+  const finalSlashIndex = path.lastIndexOf('/')
```

Comment:
Would be nice if we could use `path.dirname` in here rather than rolling our own...

Reply:
I didn't end up needing this it seems (it was used in the two transform iterations I attempted prior), so I'll bin it off.

Reply:
Fixed in 828a4a3